src/etc/inc/notices.inc

```diff
@@ -421,11 +421,11 @@ function send_smtp_message($message, $subject = "(no subject)", $force = false)
 
 function notify_via_telegram($message, $force = false) {
 	global $config;
-	if ((!isset($config['notifications']['telegram']['enabled']) || (!$force))
-	|| (!$config['notifications']['telegram']['api'] || !$config['notifications']['telegram']['chatid'])) {
-		if($force) {
-			$err_msg="Unable to test Telegram notification without both API Key & Chat ID set";
-			return $err_msg;
+	if ((!isset($config['notifications']['telegram']['enabled']) && (!$force))
+			|| !$config['notifications']['telegram']['api']
+			|| !$config['notifications']['telegram']['chatid']) {
```

Comment:
The indentation here still isn't right. It should be:

```
<tab>if ([...]
<tab><four spaces>|| ([...]
```

There are examples at the link I mentioned before: ["K&R, BSD KNF variant style" formatting](https://www.netgate.com/docs/pfsense/development/developer-style-guide.html#indent-style).

Reply:
Got it - thanks for taking the time to explain.